diff -r 49c52f767f79 guava/src/main/java/com/fasterxml/jackson/datatype/guava/GuavaModule.java
--- a/src/main/java/com/fasterxml/jackson/datatype/guava/GuavaModule.java	Wed Nov 14 19:36:36 2018 +0300
+++ b/src/main/java/com/fasterxml/jackson/datatype/guava/GuavaModule.java	Wed Nov 14 19:37:05 2018 +0300
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.core.Version;
 
 import com.fasterxml.jackson.databind.*;
+import com.fasterxml.jackson.databind.Module;
 import com.fasterxml.jackson.datatype.guava.ser.GuavaBeanSerializerModifier;
 import com.google.common.collect.BoundType;
 
